homeassistant/components/eight_sleep/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]: {error}"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]: {error}"
```

Comment:
I should point out we will be leaking english errors after translated strings.  I'm not sure there is anything that can be done about that though.

Usually we don't embed keys.  I'm not sure the logic for it but every time I've seen it done there is a followup request to remove it.

Reply:
so are you suggesting I change this to `"{error}"` instead?

Reply:
Either of

`"cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"`

or if you want to do a custom one, don't include the `[%key:common::config_flow::error::cannot_connect%]` inside.  
ex:

`"cannot_connect": "Cannot connect to eight sleep cloud: {error}"`

I expect the motivation for that translations since `[%key:common::config_flow::error::cannot_connect%]: {error}` probably doesn't make sense for RTL languages